administration/src/project-configs/bayern/dataPrivacyBase.tsx

```diff
@@ -449,11 +449,16 @@ export const DataPrivacyBaseText = () => (
       Informationen, die Sie unverschlüsselt per E-Mail an uns senden können, möglicherweise auf dem Übertragungsweg von
       Dritten gelesen werden.{' '}
     </p>
-    <p style={{ marginTop: '0pt', marginBottom: '8pt' }}>
+    <div>
       <span>
         <b>Weitergabe der Daten an externe Dritte</b>
       </span>
-    </p>
-    <p style={{ marginTop: '0pt', marginBottom: '8pt' }} />
+    </div>
+  </div>
+)
+
+export const DataPrivacyAdditionalBaseText = () => (
+  <div>
```

Comment:
Should we make this a p? Or is some manual margin necessary?

Reply:
we could add some bottom margin, but on the other hand if the region specific text gonna be added there won't be any margin neither. I think it's fine as it is.

<img width="823" alt="image" src="https://github.com/digitalfabrik/entitlementcard/assets/37902063/8df3cbd6-e237-4b5c-b406-6b0cb12a7a48">